CHANGELOG.md

```diff
@@ -16,6 +29,7 @@
 
 **Merged pull requests:**
 
+- Release 2.1.0 [\#118](https://github.com/voxpupuli/puppet_webhook/pull/118) ([dhollinger](https://github.com/dhollinger))
```

Comment:
We typically skip these. Is it intentional that these are all visible again?

Reply:
Nope. I'm working on that.

Reply:
@ekohl Should be fixed now. Looks like it also cleaned up some other issues that shouldn't have been in the changelog